### PR TITLE
added seasonal artifact display enhancements

### PR DESCRIPTION
**Pull request by mjlemay**
This continues some previous work to the ArtifactUnlocks overlay in support of https://github.com/DestinyItemManager/DIM/issues/8716:
- adds a new row to display the `reset` and `pointsUsed` values
- updates artifact unlock table styles to mimic more of the current seasonal presentation
- adds a highlight style to the tier column if the tier value for `isUnlocked` is true
- adds `pointsUsed` location key and default value for the l18n.json file

### CODE REVIEW

File: src/app/vendors/single-vendor/ArtifactUnlocks.tsx [hunk 1]
```diff
@@ -29,32 +30,44 @@ export default function ArtifactUnlocks({ characterId }: { characterId: string }
...
-                </ItemPopupTrigger>
-              ),
-          )}
-        </div>
-      ))}
-    </div>
+    <>
+      <div className={styles.stat}>
+        <div>{t('Progress.PointsUsed_plural', { count: pointsUsed })}</div>
+        <div>{t('Progress.Resets_plural', { count: resetCount })}</div>
```
Comment (delphiactual) [inline src/app/vendors/single-vendor/ArtifactUnlocks.tsx:39]:
```suggestion
        <div>{t('Progress.PointsUsed', { count: pointsUsed })}</div>
        <div>{t('Progress.Resets', { count: resetCount })}</div>
```

File: package-lock.json
Comment (delphiactual) [inline package-lock.json:1]: we are no longer using `yarn` please remove and do a `pnpm install`

**Comment by delphiactual** [comment]
did you rerun `pnpm i18n`?

File: config/i18n.json [hunk 2]
```diff
@@ -1040,6 +1040,7 @@
     "Items": "Quest Items",
     "Milestones": "Milestones & Challenges",
     "PercentPrestige": "{{pct}}% to reset",
+    "PointsUsed_plural": "{{count}} points used",
```
Comment (delphiactual) [inline config/i18n.json]:
```suggestion
    "PointsUsed": "1 point used",
    "PointsUsed_plural": "{{count}} points used",
```

**Review by delphiactual — CHANGES_REQUESTED** [review]

**Review by delphiactual — APPROVED** [review]

**Comment by delphiactual** [comment]
image

**Comment by bhollis** [comment]
@delphiactual thanks for adding screenshots. @mjlemay would you mind updating the PR description to describe what you've changed?

**Review by nev-r — APPROVED** [review]

**Review by bhollis — APPROVED** [review]
Thanks!

**Pull request merged**